packages/styles/scss/internal/content-item/_content-item.scss

```diff
@@ -118,4 +118,18 @@
   :host(#{$c4d-prefix}-content-item) ::slotted([slot='footer']) {
     margin-inline-start: 0;
   }
+
+  :host(#{$c4d-prefix}-content-item) ::slotted([slot='footer']:first-of-type) {
+    padding-top: $spacing-05;
+  }
+
+  :host(#{$c4d-prefix}-content-item) ::slotted([slot='footer']:last-of-type) {
+    padding-bottom: $spacing-07;
+  }
+
+  :host(#{$c4d-prefix}-content-item)
+    ::slotted([slot='footer']:not(:first-of-type):not(:last-of-type)) {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
```

Comment:
@m4olivei 

So, it turns out when we have multiple CTAs stacked, it won't multiple <a> elements but the entire parent, in storybook it will be `c4d-link-with-`icon and in AEM it will be `div.cds--footer--cta-link`

Therefore, each CTA will have much space between them. 

In this approach I'm still selecting `[slot='footer']` but applying the margin depending on the slotted element being the first or last in his context. If we only have 1 CTA, the browser seems to be considering it as the first and the last element - applying the proper top and bottom margin. This one will reset padding of CTAs in the middle ` ::slotted([slot='footer']:not(:first-of-type):not(:last-of-type))`

I appreciate your thoughts. Thanks @m4olivei !

Reply:
That's a clever approach! I wonder if it would be more straightforward to set the padding on the wrapper `<div>`? eg. this one: https://github.com/bruno-amorim/carbon-for-ibm-dotcom/blob/fix/content-item-cta-spacing/packages/web-components/src/components/content-item/content-item.ts#L143-L146

Have you tried that?

Reply:
@m4olivei thanks for your suggestion! Can you please check it now?